plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -353,21 +351,20 @@ class WebMappingService implements InitializingBean
          location: bboxMidpoint
       ]
 
-      if(addLocation)
-      {
-        logParams.location = bboxMidpoint
```

Comment:
We use this for our metrics and heatmap. Did this get removed entirely?

Reply:
looks to me like just the addLocation boolean got removed, which seems reasonable. Why would you not want it logged always? the bboxMidpoint seems to still be calculated. Although I have a question about that, which I'll mention in the code

Reply:
Nope, yo code was duped